packages/client/src/flow/getUserFUSDBalance.tx.js

```diff
@@ -0,0 +1,12 @@
+export const GET_USER_FUSD_BALANCE = `
```

Comment:
Add tests for newly added scripts and transactions

Reply:
Will do!